arduino/netsender/NetSender.cpp

```diff
@@ -25,20 +25,43 @@
 #include <limits.h>
 #include <string.h>
 #include <ctype.h>
+
+#define ESP32
```

Comment:
I think there is no need to manually set this depending on the platform; ```ESP32``` and ```ESP8266``` should be predefined macros from what I've read online, and then re linux, you should be able to check for ```__linux__``` assuming you're using GCC.

Reply:
I think you're right. And, yes, I use gcc when compiling without Arduino.